certification_organism/data/ir_model_fields.xml

```diff
@@ -0,0 +1,563 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="certification_organism.filename_for_x_studi_e4a6f7a0-7955-48a2-ab1c-a295727bd488" model="ir.model.fields" context="{'studio': True}">
+        <field name="copied" eval="True"/>
+        <field name="field_description">Filename for x_studio_binary_field_465_1h8jpmjfh</field>
+        <field name="model">x_project_task_worksheet_template_2</field>
```

Comment:
you can remove the field model, as in the create of ir.model.field the value of model is set based on the model_id